sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -95,9 +103,7 @@ public async Task VerifyImdsRequestWithClientIdMockAsync()
             Assert.AreEqual(request.Uri.Path, "/metadata/identity/oauth2/token");
             Assert.IsTrue(query.Contains("api-version=2018-02-01"));
             Assert.IsTrue(query.Contains($"resource={Uri.EscapeDataString(ScopeUtilities.ScopesToResource(MockScopes.Default))}"));
-            Assert.IsTrue(request.Headers.TryGetValue("Metadata", out string metadataValue));
```

Comment:
Why don't we set this header as true anymore?

Reply:
because some of these tests are now just observing the probe request, so that validation is not relevant for the purpose of the test.